tests/lms/validation/_canvas_test.py

```diff
@@ -0,0 +1,246 @@
+from abc import ABCMeta
+import json
+from unittest import mock
+
+import pytest
+import requests
+
+from lms.validation import (
+    CanvasListFilesResponseSchema,
+    CanvasPublicURLResponseSchema,
+    ValidationError,
+)
+
+
+class CommonResponseSchemaTests(metaclass=ABCMeta):
```

Comment:
Making this an ABC means that pytest doesn't try to run this test class directly

Reply:
How does that work? Does pytest have specific logic to recognize that a class uses `ABCMeta` or does it happen via some other mechanism?

Reply:
Ah no, I was mistaken, it's because the class name doesn't match the test discovery class name pattern (doesn't start with `Test`). I removed the ABC stuff cos it's actually probably more confusing than semantic